init/cli/stop.go

```diff
@@ -95,7 +95,7 @@ func stopService(ctx cli.Context, procs map[string]*os.Process) error {
 }
 
 func waitForServiceToStop(ctx cli.Context, procs map[string]*os.Process) error {
-	const numSecondsToWait = 240
+	const numSecondsToWait = 120
```

Comment:
It might be safer to give a little more time to get in there before the deployment infra does? Like 115 seconds? Otherwise if the deployment infra is a little faster with the sigkill than the sigterm we have the same problem?